core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -27,7 +27,8 @@ oppia.directive('answerGroupEditor', [function() {
       getOnSaveAnswerGroupFeedbackFn: '&onSaveAnswerGroupFeedback',
       getOnSaveAnswerGroupRulesFn: '&onSaveAnswerGroupRules',
       outcome: '=',
-      rules: '='
+      rules: '=',
+      suppressWarnings: '='
```

Comment:
suggest &, since we don't expect this value to be modified within the directive (so we don't really need two-way binding)